tt_metal/llrt/llrt.cpp

```diff
@@ -2,19 +2,38 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+#include <cassert>
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <cstdlib>
+#include <condition_variable>
+#include <functional>
+#include <mutex>
+#include <memory>
```

Comment:
Nearly, but not quite collated -- doesn't clang-format get this for you?

Reply:
I got them from the tool "include what you use" ... I think I had sorting disabled, because folks have mentioned that include file order can have impact on our builds.

You would prefer alphabetical order here, you are saying?

Reply:
I think there should no situation when you need to have a specific order. And this issues should be fixed. But it is out of context of this pr.